### ISSUE TITLE
Cloning a Neo field loses all Neo tab content

### ISSUE THREAD

**Issue by scottgib** [engram-design/FieldManager#78]
After a clone all the neo blocks are empty, I've tried with a really simple field with only one tab and one field.

Craft version: 3.6.16
Field manager: 2.2.3
NEO: 2.9.10

**Comment by engram-design** [comment]
Looks like this is a recent Neo update causing this, as I've just updated to 2.9.11 to find this issue (Previously on 2.8.14 where this was fine).

The problem is these lines: https://github.com/spicywebau/craft-neo/blob/2af37c2e9bc10e69c01c0b1ffab01cdfeba3cbf0/src/Field.php#L603-L608

Which override the block types that we setup when cloning a field, with ones prepped from POST - which is this case just doesn't exist.

Refer to the issue https://github.com/spicywebau/craft-neo/issues/462 and the additional commit on our end to address this https://github.com/verbb/field-manager/commit/19a70ed8b920278599aa03b1a32603c1bcca84f8

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/field-manager` requirement in `composer.json` to:

```
"require": {
  "verbb/field-manager": "dev-craft-3 as 2.2.4",
  "...": "..."
}
```

Then run `composer update`.

However, do note that we're also waiting on https://github.com/spicywebau/craft-neo/pull/489

**Comment by engram-design** [comment]
Fixed in [2.2.5](https://github.com/verbb/field-manager/releases/tag/2.2.5)
